core/core_bind.cpp

```diff
@@ -1034,6 +1039,8 @@ void Geometry3D::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_closest_point_to_segment_uncapped", "point", "s1", "s2"), &Geometry3D::get_closest_point_to_segment_uncapped);
 
+	ClassDB::bind_method(D_METHOD("get_triangle_barycentric_coords", "point", "v0", "v1", "v2"), &Geometry3D::get_triangle_barycentric_coords);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("get_triangle_barycentric_coords", "point", "a", "b", "c"), &Geometry3D::get_triangle_barycentric_coords);
```
To match the two methods just below, which is pretty much established here

Reply:
Done.